sources/web/datalab/polymer/polymer.json

```diff
@@ -9,7 +9,10 @@
     "components/datalab-sidebar/datalab-sidebar.html",
```

Comment:
These are elements that aren't referenced directly in the markup, so they're not discoverable by the polymer build analyzer, but that we know we're going to serve them at some point using `importHref`.